source/development_process/1_development.rst

```diff
@@ -150,11 +147,11 @@ install -r requirements.txt``. Once finished with the virtual environment, run
 `python guide <http://docs.python-guide.org/en/latest/dev/virtualenvs/>`_.
 
 
-Create the projects default branch (if not master):
+Create the project's default branch (if not master):
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
You may need to add another ~ to this line, I'm not sure if ReST will care, but it will look neater at least.